### DISCUSSION TITLE
Failing trying to import Mimir dashboard in Grafana

### DISCUSSION THREAD

**Discussion by deggja** [grafana/mimir#1667] [Help and support] [answered]
Hi,

When I add Grafana Mimir preconfigured dashboards with json I get the following error:

![image](https://user-images.githubusercontent.com/15778492/162581505-835a4741-d555-4378-80b2-1d0b2da5af3f.png)

Anyone know how to fix this?

**Comment by pracucci** [comment, marked as answer]
Grafana Mimir dashboards require a Grafana datasource of type "Prometheus" correctly configured and working. The error you get is "Failed creating data source proxy" and my gut feeling is that there's an issue with the data source.

Questions:
- Is the data source configured in Grafana?
- If you try to manually run a query (any query, like `count(up)`) using that data source in **Grafana Explore** does it succeed?

**Reply by deggja** [reply]
Thanks for the answer @pracucci

The data source was not configured correctly. It is working as expected now.
